lfsapi/lfsapi.go

```diff
@@ -51,8 +51,8 @@ func (c *Client) Context() lfshttp.Context {
 // server is not using an SSH remote or the git-lfs-transfer style of SSH
 // remote.
 func (c *Client) SSHTransfer(operation, remote string) *ssh.SSHTransfer {
-	endpoint := c.Endpoints.RemoteEndpoint(operation, remote)
-	if len(endpoint.SSHMetadata.UserAndHost) > 0 {
+	endpoint := c.Endpoints.Endpoint(operation, remote)
+	if len(operation) > 0 && len(endpoint.SSHMetadata.UserAndHost) > 0 {
```

Comment:
I'll make that change.  The reason we want to use `Endpoint` over `RemoteEndpoint` is because we want to use `lfs.url`.  I'll update the original patch with a suitable commit message.